.github/workflows/kind-e2e.yaml

```diff
@@ -17,12 +17,11 @@ jobs:
       fail-fast: false # Keep running if one leg fails.
       matrix:
         k8s-version:
-        - v1.22.x
         - v1.23.x
         - v1.24.x
         # Can not test with v.25.x yet due to PSP
```

Comment:
This comment should probably be removed, no?

Reply:
Done.